flink-table/flink-sql-client/src/test/resources/test-sql-client-defaults.yaml

```diff
@@ -137,19 +137,10 @@ catalogs:
 execution:
   planner: "$VAR_PLANNER"
   type: "$VAR_EXECUTION_TYPE"
-  time-characteristic: event-time
```

Comment:
Do not remove entries for the existing test files. The options are still supported, so the tests are valid. 
Please add a new testing yaml to verify the default behaivor.